### PR TITLE
fix makefile init to .init and add virtualenvs.in-project = true

### PR DESCRIPTION
**Pull request by Tseian**

### CODE REVIEW

File: poetry/poetry.lock [hunk 1]
```diff
@@ -48,34 +48,35 @@ test = ["flake8 (==3.7.8)", "hypothesis (==3.55.3)"]
 
 [[package]]
 name = "dnspython"
-version = "2.2.1"
+version = "2.3.0"
 description = "DNS toolkit"
 category = "dev"
 optional = false
-python-versions = ">=3.6,<4.0"
+python-versions = ">=3.7,<4.0"
```
Comment (adriangb) [inline poetry/poetry.lock]: This seems unrelated / unnecessary, why does the lockfile need to be updated?

File: poetry/workspaces/app/src/namespace/app/test_main.py [hunk 2]
```diff
@@ -1,2 +1,3 @@
 def test_import():
     from namespace.app.main import main  # type: ignore
+    main()
```
Comment (adriangb) [inline poetry/workspaces/app/src/namespace/app/test_main.py]:
```suggestion
    main(1)
```

File: poetry/workspaces/app/src/namespace/app/main.py [hunk 3]
```diff
@@ -6,9 +6,8 @@
 
 
 def main() -> None:
```
Comment (adriangb) [inline poetry/workspaces/app/src/namespace/app/main.py]:
```suggestion
def main(max_iters: float | None) -> None:
```

File: poetry/workspaces/app/src/namespace/app/main.py [hunk 4]
```diff
@@ -6,9 +6,8 @@
...
-    while True:
-        print(orjson.dump(Foo(name="app").dict()).decode())
-        sleep(1)
+    print(orjson.dumps(Foo(name="app").dict()).decode())
+    sleep(1)
```
Comment (adriangb) [inline poetry/workspaces/app/src/namespace/app/main.py]:
```suggestion
    iter = 0
    if max_iters is None:
        max_iters = float("inf")
    while iter < max_iters:
        print(orjson.dumps(Foo(name="app").dict()).decode())
        sleep(1)
```

File: poetry/poetry.lock (hunk 1 above)
Comment (Tseian) [reply poetry/poetry.lock]: Accidentally deleted the lock file.

**Review by adriangb — APPROVED** [review]
